File: src/main/java/com/distribution/modules/memeber/entity/WithdrawalInfo.java
package com.distribution.modules.memeber.entity;

import lombok.Data;

<<<<<<< HEAD
=======
import javax.validation.constraints.NotNull;
>>>>>>> origin/develop
import java.math.BigDecimal;

/**
 * 提现信息表
<<<<<<< HEAD
 * 
 * @Auther: liuxinxin
 * @Date: 2018/5/22 22:01
 * @Description: 
=======
 *
 * @Auther: liuxinxin
 * @Date: 2018/5/22 22:01
 * @Description:
>>>>>>> origin/develop
 */
@Data
public class WithdrawalInfo {

    private String id;
    private String withdrawNum;
<<<<<<< HEAD
    private String withdrawType;
    private String withdrawName;
    private String withdrawMobile;
    private String withdrawCard;
    private BigDecimal withdrawAmount;
    private BigDecimal withdrawPoundage;
    private BigDecimal realAmount;
=======
    /**
     * 提现状态 0:失败,1:成功
     */
    private String withdrawType;
    /**
     * 提现金额
     */
    @NotNull
    private BigDecimal withdrawAmount;
    private String withdrawName;
    @NotNull
    private String withdrawMobile;
    private String withdrawCard;
    private BigDecimal withdrawPoundage;
    private BigDecimal realAmount;
    private String aliPayAccount;
>>>>>>> origin/develop
    private String isDelete;
    private String addTime;
    private String updateTime;
}

File: src/main/kotlin/com/distribution/modules/memeber/entity/WithdrawalInfo.kt
package com.distribution.modules.memeber.entity

import java.math.BigDecimal
import javax.validation.constraints.NotNull

/**
 * 提现信息表
 *
 * @Auther: liuxinxin
 * @Date: 2018/5/22 22:01
 * @Description:
 */

data class WithdrawalInfo(
        var id: String? = null,

        var withdrawNum: String? = null,
        /**
         * 提现状态 0:失败,1:成功
         */
        var withdrawType: String? = null,
        /**
         * 提现金额
         */
        @NotNull
        var withdrawAmount: BigDecimal? = null,
        var withdrawName: String? = null,
        @NotNull
        var withdrawMobile: String? = null,
        var withdrawCard: String? = null,
        var withdrawPoundage: BigDecimal? = null,
        var realAmount: BigDecimal? = null,
        var aliPayAccount: String? = null,
        var isDelete: String? = null,
        var addTime: String? = null,
        var updateTime: String? = null
)
